avocado_vt/plugins/vt_settings.py

```diff
@@ -16,18 +16,14 @@
 Avocado plugin that extends the settings path of our config paths
 """
 
-import os
+import importlib.resources
 
 from avocado.core.plugin_interfaces import Settings
-from pkg_resources import resource_filename, resource_listdir
 
 
 class VTSettings(Settings):
     def adjust_settings_paths(self, paths):
-        base = resource_filename("avocado_vt", "conf.d")
-        for path in [
-            os.path.join(base, conf)
-            for conf in resource_listdir("avocado_vt", "conf.d")
-            if conf.endswith(".conf")
-        ]:
-            paths.insert(0, path)
+        base = importlib.resources.files("avocado_vt").joinpath("conf.d")
+        for path in base.iterdir():
+            if path.is_file() and path.name.endswith(".conf"):
+                paths.insert(0, path)
```

Comment:
```suggestion
                paths.insert(0, str(path))
```
I think that we could keep the type of path as the raw filesystem path for consistency.
Before:
```
>>> settings.all_config_paths
['/root/kar/workspace/avocado-vt/avocado_vt/conf.d/vt_joblock.conf', '/root/kar/workspace/avocado-vt/avocado_vt/conf.d/vt.conf', '/root/kar/workspace/etc/avocado/avocado.conf', '/root/kar/workspace/etc/avocado/conf.d/plugins.conf', '/root/kar/workspace/etc/avocado/conf.d/vt.conf', '/root/kar/workspace/etc/avocado/conf.d/vt_joblock.conf']
```
Now:
```
>>> settings.all_config_paths
[PosixPath('/root/kar/workspace/avocado-vt/avocado_vt/conf.d/vt_joblock.conf'), PosixPath('/root/kar/workspace/avocado-vt/avocado_vt/conf.d/vt.conf'), '/root/kar/workspace/etc/avocado/avocado.conf', '/root/kar/workspace/etc/avocado/conf.d/plugins.conf', '/root/kar/workspace/etc/avocado/conf.d/vt.conf', '/root/kar/workspace/etc/avocado/conf.d/vt_joblock.conf']

```

Reply:
Although the invoke in avocado will handle this, but I can accept this, keeping type consistent can avoid unexpected problems.